kinesis/kinesis.go

```diff
@@ -185,6 +189,32 @@ func (outputPlugin *OutputPlugin) Flush() error {
 	return outputPlugin.sendCurrentBatch()
 }
 
+func replaceDots(obj map[interface{}]interface{}) map[interface{}]interface{} {
+	newObj := make(map[interface{}]interface{})
```

Comment:
I'd like this code to work without requiring an additional map allocation... that's going to add a lot overhead.